guides/Shopify.md

```diff
@@ -0,0 +1,55 @@
+# Deploying to Shopify with CircleCI
+
+## The Challenge:
+
+The primary challenge we faced when developing for Shopify was that the codebase checked in to Git was getting out of sync with the codebase deployed to Shopify. This issue was compounded when using the same theme on multiple stores e.g. for simple staging + production environments or in more complex, multi-geo scenarios.
+
+## The Solution:
+
+To combat the issue, we configured the CI server, in this case CircleCI, to manage all of the deployments for us. We hooked the master branch to deploy direct to the production store and the staging branch to the staging store.
+
```

Comment:
Let's add a title 'Tools' here and link to Circle, Grunt and the Grunt shopify library